contracts/Airdrop.sol

```diff
@@ -3,10 +3,11 @@ pragma solidity ^0.7.0;
 
 import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 
-contract AirDrop {
 
-    bytes32 _rootHash;
-    IERC20 _token;
+contract Airdrop {
+
+    bytes32 private _rootHash;
```

Comment:
why make this private and write a getter function, just make it public

Reply:
added 52ae30d